relay-server/src/service.rs

```diff
@@ -147,12 +149,10 @@ impl ServiceState {
         let upstream_relay = UpstreamRelay::new(config.clone());
         registry.set(Arbiter::start(|_| upstream_relay));
 
-        let outcome_producer = OutcomeProducer::create(config.clone())?;
-        let outcome_producer = Arbiter::start(|_| outcome_producer);
-        registry.set(outcome_producer.clone());
+        // TODO(tobias): Check if this is good enough or if we want this to have its own tokio runtime?
```

Comment:
The Kafka producer used to consume a fair amount of CPU and introduced delays for other services sharing the same runtime in the past. We can benchmark this again at a later point.

For now, let's create a dedicated runtime for the outcome producer. Feel free to add the `OutcomeAggregator` into that same runtime, though.